api/api_test.go

```diff
@@ -173,111 +173,6 @@ func TestHealthCheck(t *testing.T) {
 	runHandlerTestCases(t, "health check", HealthCheckHandler, healthCheckCases, true)
 }
 
-func TestServerReadOnlyRoutes(t *testing.T) {
```

Comment:
This became useless and since we're going to remove readOnly anyways I simply moved it out of the way completely.